tensorboard/components/tf_backend/test/backendTests.ts

```diff
@@ -16,13 +16,26 @@ limitations under the License.
 import {filterTags, getRuns, getTags, RunToTag, TYPES} from '../backend.js';
 import {RequestManager} from '../requestManager.js';
 import {createRouter, setRouter} from '../router.js';
-import {queryEncoder} from '../urlPathHelpers.js';
+import {addParams} from '../urlPathHelpers.js';
 
 describe('urlPathHelpers', () => {
-  it('queryEncoder works on spaces and parens', () => {
-    const params = {foo: 'something with spaces and (parens)'};
-    const actual = queryEncoder(params);
-    const expected = 'something+with+spaces+and+%28parens%29';
+  it('addParams leaves input untouched when there are no parameters', () => {
```

Comment:
The tests seem to make sense to me. Yeah, we really ought to get them running again.

Reply:
Agreed.